### PR TITLE
Add pinned repos from terraform-providers

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
cc @shuaibiyy

File: README.md [hunk 1]
```diff
@@ -8,19 +8,22 @@ Terraform enables you to safely and predictably create, change, and improve prod
...
-* [Community Modules](#community-modules)
-* [Private Module Registries](#private-module-registries)
-* [Providers](#providers)
-* [Testing](#testing)
-* [Tools](#tools)
-* [Libraries](#libraries)
-* [Terraform Enterprise](#terraform-enterprise)
-* [Talks](#talks)
-* [Editor Plugins](#editor-plugins)
+- [Awesome Terraform ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-terraform-awesomehttpsgithubcomsindresorhusawesome)
```
Comment (shuaibiyy) [inline README.md]: I guess the only change needed is the addition of the license? The awesome terraform logo is already there.

Comment (chenrui333) [reply README.md]: This must be brought in by my editor, sorry about that. I will fix this.

**Comment by chenrui333** [comment]
@shuaibiyy I made some updates and also sort out the provider lines. Sorry for early undesired changes from editor. 😄

**Comment by shuaibiyy** [comment]
Thanks for the additions!

**Pull request merged**